packages/renderer/src/lib/pod/PodDetailsSummary.svelte

```diff
@@ -2,13 +2,20 @@
 import type { V1Pod } from '@kubernetes/client-node';
 import { onMount } from 'svelte';
 
+import { kubernetesCurrentContextEvents } from '/@/stores/kubernetes-contexts-state';
+
 import KubePodDetailsSummary from '../kube/KubePodDetailsSummary.svelte';
 import type { PodInfoUI } from './PodInfoUI';
 import PodmanPodDetailsSummary from './PodmanPodDetailsSummary.svelte';
 
-export let pod: PodInfoUI;
-let kubePod: V1Pod | undefined;
-let getKubePodError: string;
+interface Props {
+  pod: PodInfoUI;
+}
+
+let { pod }: Props = $props();
+let kubePod: V1Pod | undefined = $state();
```

Comment:
question: is it a better solution to use/follow $state() to initialize a value to undefined ? Rather than $state(undefined)